src/frameworks/flutter/index.ts

```diff
@@ -50,9 +50,13 @@ export function init(setup: any, config: any) {
   return Promise.resolve();
 }
 
-export function build(cwd: string): Promise<BuildResult> {
+export async function build(cwd: string): Promise<BuildResult | undefined> {
```

Comment:
the return type didn't change

```suggestion
export async function build(cwd: string): Promise<BuildResult> {
```